index.js

```diff
@@ -59,4 +59,182 @@ function hideNav() {
+    const textbtn2 = document.createElement('p');
+
+    closeImg.src = 'images/HideNavIcon.png';
+    imagePopUp.src = 'images/ImagePopUp.png';
+    liveIcon.src = 'images/IconLivePopUp.png';
+    githubIcon.src = 'images/GithubPopUp.png';
+
+    btnsList.classList.add('btns_List');
+    closeImg.classList.add('close_popUp');
+    closePopUp.classList.add('btn1_popUp');
+    btn1PopUp.classList.add('btn1_popUp');
+    btn2PopUp.classList.add('btn1_popUp');
+    textPopUp.classList.add('p_popUp');
+    titlePopUp.classList.add('h1_popUp');
+    imagePopUp.classList.add('img_popUp');
+    githubIcon.classList.add('icon_popUp');
+    liveIcon.classList.add('icon_popUp');
+    containerPopUp.classList.add('container_popUp');
+    AlignFlexImg.classList.add('Align_FlexImg');
+    textPopUpDiv.classList.add('q1');
+    btn1P.classList.add('q2');
+    btn2P.classList.add('q2');
+    btn3P.classList.add('q2');
+    ulBtnP.classList.add('ul_btnP');
+    textbtn1.classList.add('text_btn');
+    textbtn2.classList.add('text_btn');
+    titleAndIcon.classList.add('titile_and_icon');
+
+    body.appendChild(containerPopUp);
+    containerPopUp.append(titleAndIcon, ulBtnP, AlignFlexImg);
+    titleAndIcon.append(closeImg, titlePopUp);
+    ulBtnP.append(btn1P, btn2P, btn3P);
+    AlignFlexImg.append(imagePopUp, textPopUpDiv);
+    textPopUpDiv.append(textPopUp, btnsList);
+    btnsList.append(btn1PopUp, btn2PopUp);
+    btn1PopUp.append(textbtn1, liveIcon);
+    btn2PopUp.append(textbtn2, githubIcon);
+
+    btn1P.innerText = 'Html';
+    btn2P.innerText = 'Bootstrap';
+    btn3P.innerText = 'Ruby on Rails';
+    textbtn1.textContent = 'See live';
+    textbtn2.textContent = 'See Source';
+    textPopUp.innerText = projectDisplay.popUpText;
+    titlePopUp.innerText = projectDisplay.title;
+
+    section1.style.filter = 'blur(8px)';
+    MainSectionproject.style.filter = 'blur(8px)';
+    sectionAbout.style.filter = 'blur(8px)';
+    contactForm.style.filter = 'blur(8px)';
+    NavAll2.style.filter = 'blur(8px)';
+    NavAll1.style.filter = 'blur(8px)';
+
+    NavAll1.style.pointerEvents = 'none';
+    NavAll2.style.pointerEvents = 'none';
+    section1.style.pointerEvents = 'none';
+    MainSectionproject.style.pointerEvents = 'none';
+    sectionAbout.style.pointerEvents = 'none';
+    contactForm.style.pointerEvents = 'none';
+
+    closeImg.addEventListener('click', () => {
+      containerPopUp.style.display = 'none';
+      section1.style.filter = 'none';
+      MainSectionproject.style.filter = 'none';
+      sectionAbout.style.filter = 'none';
+      contactForm.style.filter = 'none';
+      NavAll1.style.filter = 'none';
+      NavAll2.style.filter = 'none';
+      NavAll1.style.filter = 'all';
+      NavAll2.style.filter = 'all';
+      section1.style.pointerEvents = 'all';
+      MainSectionproject.style.pointerEvents = 'all';
+      sectionAbout.style.pointerEvents = 'all';
+      contactForm.style.pointerEvents = 'all';
+    });
+  });
```

Comment:
- [ ] One of the purposes of this project is for you to get used to working with objects and arrays and display the information from them dynamically on your document. So instead of creating each element by hand, please create a different object that can hold all the necessary information, and then create a function that iterates over them and display the information on the screen. 
Your objects may look something like this:
```
const projects = {
  title: "my project title",
  description: "my project description"
}
```
And you can have many objects inside another object. Then all you need to do is iterate over your objects and display the information on the screen, good luck!